### ISSUE TITLE
Some options are not properly backed

### ISSUE THREAD

**Issue by wheremyfoodat** [grumpycoders/pcsx-redux#1111]

**Comment by buruto** [comment]
> * [❌] Window settings (Maximization and x/y/w/h) seem to not persist sometimes as per [All window coordinates including both position and size are both discarded and lost after maximize and reboot #1110](https://github.com/grumpycoders/pcsx-redux/issues/1110)
> * [❌] Memory editor settings don't persist as per [The "Memory Editor #1" tab is missing every time I reboot PCSX-Redux and I have to bring it back over and over again #1108](https://github.com/grumpycoders/pcsx-redux/issues/1108) and [All the Options of Memory Editor #1 are not saved to and loaded from a file or disk on close, open or reboot of PCSX-Redux #1092](https://github.com/grumpycoders/pcsx-redux/issues/1092)
> * [❌] Assembly window settings don't persist as per [All the Assembly Options are not saved to and loaded from a file or disk on close, open or reboot of PCSX-Redux #1091](https://github.com/grumpycoders/pcsx-redux/issues/1091)

Are these some issues that you are working on them?

**Comment by johnbaumann** [comment]
One setting that might need addressed in this issue is the m_fullscreen variable of gui.h I don't believe this setting is saved, and it might be beneficial to treat the window position variables as separate if m_fullscreen = true.

As is, if you exit the emulator while in full screen mode, a reboot opens in a windowed view that appears fullscreen - as in, the emulator's Title Bar and sizing border are hidden/stretched outside of the viewable area.

**Comment by wheremyfoodat** [comment]
> > * [❌] Window settings (Maximization and x/y/w/h) seem to not persist sometimes as per [All window coordinates including both position and size are both discarded and lost after maximize and reboot #1110](https://github.com/grumpycoders/pcsx-redux/issues/1110)
> > * [❌] Memory editor settings don't persist as per [The "Memory Editor #1" tab is missing every time I reboot PCSX-Redux and I have to bring it back over and over again #1108](https://github.com/grumpycoders/pcsx-redux/issues/1108) and [All the Options of Memory Editor #1 are not saved to and loaded from a file or disk on close, open or reboot of PCSX-Redux #1092](https://github.com/grumpycoders/pcsx-redux/issues/1092)
> > * [❌] Assembly window settings don't persist as per [All the Assembly Options are not saved to and loaded from a file or disk on close, open or reboot of PCSX-Redux #1091](https://github.com/grumpycoders/pcsx-redux/issues/1091)
> 
> Are these three issues that you are working on them and you decided to put all of them here in a single issue? Brilliant! You are smarter than me!

All of us are a bit short on time these days so I don't think somebody is working on these at this very moment. However they should eventually be fixed. Grouping similar issues (ie issues that are of a similar nature like this one, or issues that concern a specific widget, etc) is a pretty similar practice as it avoids creating too many issues and flooding the issues tab, burying other people's issues.

It's similar to being in a classroom. You are free to speak, but it's important to make sure the other students get to be heard too.

**Comment by buruto** [comment]
> > > * [❌] Window settings (Maximization and x/y/w/h) seem to not persist sometimes as per [All window coordinates including both position and size are both discarded and lost after maximize and reboot #1110](https://github.com/grumpycoders/pcsx-redux/issues/1110)
> > > * [❌] Memory editor settings don't persist as per [The "Memory Editor #1" tab is missing every time I reboot PCSX-Redux and I have to bring it back over and over again #1108](https://github.com/grumpycoders/pcsx-redux/issues/1108) and [All the Options of Memory Editor #1 are not saved to and loaded from a file or disk on close, open or reboot of PCSX-Redux #1092](https://github.com/grumpycoders/pcsx-redux/issues/1092)
> > > * [❌] Assembly window settings don't persist as per [All the Assembly Options are not saved to and loaded from a file or disk on close, open or reboot of PCSX-Redux #1091](https://github.com/grumpycoders/pcsx-redux/issues/1091)
> > 
> > 
> > Are these three issues that you are working on them and you decided to put all of them here in a single issue? Brilliant! You are smarter than me!
> 
> All of us are a bit short on time these days so I don't think somebody is working on these at this very moment. However they should eventually be fixed. Grouping similar issues (ie issues that are of a similar nature like this one, or issues that concern a specific widget, etc) is a pretty similar practice as it avoids creating too many issues and flooding the issues tab, burying other people's issues.
> 
> It's similar to being in a classroom. You are free to speak, but it's important to make sure the other students get to be heard too.

You are correct. I also realized that these issues are similar and had to be grouped. You did the right thing and I know that you are not always available.

**Comment by buruto** [comment]
> One setting that might need addressed in this issue is the m_fullscreen variable of gui.h I don't believe this setting is saved, and it might be beneficial to treat the window position variables as separate if m_fullscreen = true.
> 
> As is, if you exit the emulator while in full screen mode, a reboot opens in a windowed view that appears fullscreen - as in, the emulator's Title Bar and sizing border are hidden/stretched outside of the viewable area.

I am not playing fullscreen but only windowed. I don't even know how to play in fullscreen and if this is possible at all.
